proxy/src/avatar.rs

```diff
@@ -116,34 +123,35 @@ impl Color {
 
     /// Compute the lightness of a color.
     #[must_use]
-    pub fn lightness(&self) -> f32 {
-        let r = self.r as f32;
-        let g = self.g as f32;
-        let b = self.b as f32;
-        let n = 0xff as f32;
+    pub fn lightness(self) -> f32 {
+        let r = f32::from(self.r);
+        let g = f32::from(self.g);
+        let b = f32::from(self.b);
+        let n = 255_f32;
```

Comment:
Also exactly the same as `0xff as f32` but less clear that we're looking at the max value of a byte. Maybe `u8::MAX as f32` would be even better.